flipperdevices/Flipper-Android-App
# Save and view infrared remote control

**Background**

This PR introduces saving and opening saved infrared remote controls from mobile application.

**Changes**

- Save Remote Control
- Open Remote Control from infrared screen

**Test plan**

- Find some remote control
- Click save to save remote on grid screen
- Enter name and save it on infrared remote rename screen
- See configuring screen which will configure remote control for further usage
- Go to archive and try open saved remote control
- See the grid screen is displayed
